### DISCUSSION TITLE
Noob: ctrlr Version = 5.3.201 doesn't open bpanelz files (Ubuntu 20)

### DISCUSSION THREAD

**Discussion by antiquodian-wallah** [RomanKubiak/ctrlr#238] [Q&A]
When I File->Open Panel,   I can only see *.panel files. The *.bpanelz files are not visible. What am I doing wrong?
Many thanks!

**Comment by Godlike-Productions** [comment]
I had a similar issue trying to open a bpanelz file I made on windows in OSX the other day.  After trying several times I eventually got it to open the panel file with the full folder structure instead.  I have no idea why, but it may have something to do with the ctrlr state save data.

I don't know where it is in Linux but try to delete the config file and open ctrlr again.

**Comment by antiquodian-wallah** [comment]
I resolved it. 

Under the Edit->Preferences, uncheck the radio button labelled "Use OS native file open/save dialogs (disable for Linux)". Close/reopen Crtlr. As Yogi Berra said, "You can see a lot by looking."

Now I can open .bpanelz files on Ubuntu.

Thanks!
